Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10
+WORKDIR /usr/src/app
```

Comment:
This may prevent the app form deploying properly. This would be due to the fact you don't have a src folder. Double check via this article

https://nodejs.org/de/docs/guides/nodejs-docker-webapp/